packages/trader/src/Modules/Trading/Components/Form/TradeParams/amount.tsx

```diff
@@ -168,13 +172,15 @@ const Amount = observer(({ is_minimized, is_nativepicker }) => {
                 duration_unit={duration_unit}
                 expiry_type={expiry_type}
                 onChange={onChange}
-                value={parseInt(is_equal)}
+                value={Number(is_equal)}
                 has_equals_only={has_equals_only}
             />
             {is_multiplier && (
                 <React.Fragment>
                     <Multiplier />
                     <MultipliersInfo
+                        /* 
+                        // @ts-expect-error observer wrapped component props cant be detected until its ts-migrated */
```

Comment:
@henry-deriv will this comment fit in 1 line ? :)

Reply:
yes